Source/HTTP/httpcall.cpp

```diff
@@ -538,6 +538,28 @@ try
 }
 CATCH_RETURN()
 
+STDAPI 
+HCHttpCallGetRequestUrl(
+	_In_ hc_call_handle_t call,
+	_Out_ const char** url
+	) HC_NOEXCEPT
+try
+{
+	if (call == nullptr)
+	{
+		return E_INVALIDARG;
+	}
+
+	if (call->url.empty())
```

Comment:
data is safe to call even on empty strings, it will return a pointer to an empty c string (so just a null terminator) which may be easier to consume for the client (std::string does not play nice with null pointers)

Reply:
oh ok ill just return empty string then